assets/js/modules/analytics/components/settings/SettingsView.js

```diff
@@ -170,18 +192,39 @@ export default function SettingsView() {
 								<DisplaySetting value={ ga4MeasurementID } />
 							</p>
 						</div>
+						<div className="googlesitekit-settings-module__meta-item">
+							<h5 className="googlesitekit-settings-module__meta-item-type">
+								{ createInterpolateElement(
```

Comment:
@aaemnnosttv Note the use of `VisuallyHidden` in this pattern in the elements preceding this one

Reply:
Ah okay, yeah I missed that. In that case then it's fine and consistent with what's already there 👍